### PR TITLE
Style updating

### LINKED ISSUE #12: Style job listings
Arrange job listing component into columns and rows that are styled to match theme
Easy to gather a quick rundown on each jobs company, tech stack, and truncated job description

### PR DESCRIPTION
**Pull request by ksk5280**
Arranges content into columns.
Adds Turing colors.
Has different views for job show and index page.

@rrgayhart @brianrip @natevenn @cheljoh 

closes #12

### CODE REVIEW

**Comment by brianrip** [comment]
I like the three column layout. @ksk5280 I think some extra margin padding on the left and right side of page would look good. Also, lets even out the top and bottom margins of the individual job listings and align the search bar along the right edge to match padding of job listing. I will make the changes and see what you guys think.

**Comment by ksk5280** [comment]
@brianrip sounds good, thanks! We may also want to have the navbar stick to the top of the page. What do you think?

**Comment by cheljoh** [comment]
Nice job Kimi!

File: lib/components/JobIndex.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import Header from './header';
+import Header from './Header';
```
Comment (natevenn) [inline lib/components/JobIndex.js]: Does this mean that we are changing all of our files to begin with a capital letter? currently most of them begin with a lower case letter. I dont think it matters which way we go with it but just to make sure we are consistent.

**Comment by natevenn** [comment]
This alll looks good. I will merge this in to my testing branch and see if these changes break any of the tests

**Comment by brianrip** [comment]
Requested changes made. I would like to see this merged if possible. @natevenn @ksk5280 @cheljoh

**Comment by cheljoh** [comment]
Took a look at the branch, looks really slick! Nice job @ksk5280 and @brianrip!

**Pull request merged**